### DISCUSSION TITLE
Switch from swiftmailer to symfony mailer

### DISCUSSION THREAD

**Discussion by bambamboole** [laravel/framework#34347] [General] [answered]
Hey guys,

I just wanted to ask if there is a planned switch from Swiftmailer to Symfony's new `Mail` component.
There was a PR about a year ago: https://github.com/laravel/framework/pull/29325

Are there any blockers or do we just need some man power on this?

**Comment by driesvints** [comment, marked as answer]
Main blocker atm was that it hadn't matured enough yet. Symfony is planning on doing some more updates soon which might lead us to introduce it in Laravel 9 or 10 (still TBD).

**Reply by bambamboole** [reply]
thanx for the info @driesvints

**Comment by mfn** [comment]
Today this PR was merged https://github.com/swiftmailer/swiftmailer/pull/1352
> Swiftmailer will stop being maintained at the end of November 2021

**Edit::** official announcement: https://symfony.com/blog/the-end-of-swiftmailer

**Reply by driesvints** [reply]
Yes, we're going to look into the Symfony Mailer component.

**Comment by cwilby** [comment]
See https://github.com/laravel/framework/pull/38481
